torch/_functorch/functional_call.py

```diff
@@ -0,0 +1,79 @@
+from typing import Dict, Union, Any, Tuple
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+from torch._functorch.utils import exposed_in
+
+
+@exposed_in("torch.func")
+def functional_call(
+    module: 'torch.nn.Module',
+    parameter_and_buffer_dicts: Union[Dict[str, Tensor], Tuple[Dict[str, Tensor], ...]],
+    args: Union[Any, Tuple],
+    kwargs: Dict[str, Any] = None,
```

Comment:
Just to check: Did the weight tying update for stateless.functional_call happen already? If so, should functional_call have an option to toggle the weight tying behavior?

Reply:
#90477 hasn't been approved yet. I can add it in there if this lands first

Reply:
Sorry, totally missed that PR